src/app/map-ui/ui-slider/ui-slider.component.scss

```diff
@@ -72,4 +72,20 @@
     transition: transform 0.2s ease;
     transform: scale(1);
     
+}
+
+.el-slider.vertical {
+    transform: rotate(270deg);
```

Comment:
Instead of rotating the element to create the vertical version, I think it might be better to have a fixed width and a fluid height, so that it fills the height of it's parent container.  That way styling the component is easier, as we could specify the height styles instead of specifying width styles that will actually reflect the height of the component.

Reply:
Yeah, the transform approach was easier at first, but it'd be more complicated to maintain, works for me!